### PR TITLE
251 - refactor card add cards grid

### LINKED ISSUE #251: Add field to card component - grid set up
Cards are often used as category-esque pages (for example meet the team or issues we support: https://www.comicrelief.com/your-impact/the-issues-we-support)

We need to enable cards to be left aligned and 'overspill' onto the next line if they exceed the maximum number of cards for that line - e.g. we need to be able to have the one card on a row if there are 5 cards set at the 25% width. 

image

### PR DESCRIPTION
**Pull request by gusliedke**
fixes #251 and https://github.com/comicrelief/campaign/issues/1157

### CODE REVIEW

File: sass/base/components/card-block/_card.scss [hunk 1]
```diff
@@ -6,18 +6,16 @@
...
   position: relative;
   display: inline-block;
   overflow: hidden;
   margin-bottom: 20px;
   text-align: left;
-  @include breakpoint($screen-md-only) {
-    width: calc(100% - (4 * 1.25%)) !important;
-    margin: 1.25% calc(2 * 1.25%) !important;
-  }
+  width: calc(100% - (4 * 1.25%));
```
Comment (AndyEPhipps) [inline sass/base/components/card-block/_card.scss]: Maybe nicer to use the grid/gutter variables here, rather than hardcoded % values?

**Comment by Heleen-cr** [comment]
@gusliedke I was going to pick up #242, but noticed that you might be picking that up in this ticket?

**Comment by gusliedke** [comment]
@Heleen-cr ok I can add this tweak to this work.

**Comment by Heleen-cr** [comment]
will do #242 in a new PR.

**Comment by Heleen-cr** [comment]
passed QA

**Pull request merged**